code/drasil-printers/lib/Language/Drasil/HTML/Helpers.hs

```diff
@@ -94,6 +94,10 @@ wrapGen' sepf Id s ti _ = \x ->
   let tb c = text ("<" ++ c ++ " id=\"") <> ti <> text "\">"
       te c = text $ "</" ++ c ++ ">"
   in sepf [tb s, indent x, te s] 
+wrapGen' sepf Align s ti _ = \x ->
+  let tb c = text ("<" ++ c ++ " align=\"") <> ti <> text "\">"
```

Comment:
Since there is the repeated pattern of `let tb c = text ("<" ++ c ++ " <lowercase Variation>=\"")`, I'm wondering how much of these case could be generated, as well as if we could define a `show` for `Variation` to assist in this 🤔 Don't know how much effort this would be for its benefits though lol

Love this commit regardless!

Reply:
Improved in https://github.com/JacquesCarette/Drasil/pull/3823/commits/3614102058874ba7755b208978b52b1c005356ff.